### PR TITLE
Add redis deployment

### PR DESCRIPTION
**Pull request by acasajus**
Download, compile and run redis

### CODE REVIEW

File: deployments/redis.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Install a Redis local instance
+#
+# Add at least the following environment variables to your project configuration
+# (otherwise the defaults below will be used).
+# * REDIS_VERSION
+# * REDIS_PORT
+#
+# Include in your builds via
+# \curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/cockroachdb.sh | bash -s
```
Comment (joesiewert) [inline deployments/redis.sh]: `cockroachdb.sh` > `redis.sh`

File: deployments/redis.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+REDIS_DIR=${REDIS_DIR:="$HOME/redis"}
+
+set -e
+CACHED_DOWNLOAD="${HOME}/cache/redis-${REDIS_VERSION}.tar.gz"
+
+mkdir -p "${REDIS_DIR}"
+wget --continue --output-document "${CACHED_DOWNLOAD}" "http://download.redis.io/releases/redis-${REDIS_VERSION}.tar.gz"
+tar -xf "${CACHED_DOWNLOAD}" --directory "${REDIS_DIR}"
+
+# Make sure to use the exact parameters you want for DynamoDB and give it enough sleep time to properly start up
```
Comment (joesiewert) [inline deployments/redis.sh]: Remove or update this comment

File: deployments/redis.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#
+# Add at least the following environment variables to your project configuration
+# (otherwise the defaults below will be used).
+# * REDIS_VERSION
+# * REDIS_PORT
+#
+# Include in your builds via
+# \curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/cockroachdb.sh | bash -s
+REDIS_VERSION=${REDIS_VERSION:="4.0.2"}
+REDIS_PORT=${REDIS_PORT:="26257"}
```
Comment (joesiewert) [inline deployments/redis.sh]: `26257` > `6380`

Default Redis port is 6379, but will be in use by the older Redis version already running

File: deployments/redis.sh (hunk 1 above)
Comment (joesiewert) [inline deployments/redis.sh]: Move script from `deployments/redis.sh` to `packages/redis.sh`

**Review by joesiewert — CHANGES_REQUESTED** [review]
Thanks for the PR - just a few small changes

**Comment by acasajus** [comment]
Clearly I copied the script from the cockroachdb one 😃

**Review by joesiewert — APPROVED** [review]

**Comment by joesiewert** [comment]
🙇

**Pull request merged**